### ISSUE TITLE
3.x Drag and drop resources in the tree does not work

### ISSUE THREAD

**Issue by Ruslan-Aleev** [modxcms/revolution#14987]
### Summary
Drag and drop of resources in the tree does not work, it gives an error in the console.

### How it look like:

![res-drag](https://user-images.githubusercontent.com/12523676/73593895-1020a180-4522-11ea-834f-d511122d900f.gif)

![res-dd-2](https://user-images.githubusercontent.com/12523676/73593894-1020a180-4522-11ea-814c-e95d6978270d.png)

```
modx.tree.resource.js:758 Uncaught TypeError: Cannot read property 'Resource/Update' of undefined
    at MODx.tree.Resource._handleDrag (modx.tree.resource.js:758)
    at h.Event.fire (ext-all.js:21)
    at MODx.tree.Resource.fireEvent (ext-all.js:21)
    at Ext.tree.TreeDropZone.completeDrop (ext-all.js:21)
    at Ext.tree.TreeDropZone.processDrop (ext-all.js:21)
    at Ext.tree.TreeDropZone.onNodeDrop (ext-all.js:21)
    at Ext.tree.TreeDropZone.notifyDrop (ext-all.js:21)
    at Ext.tree.TreeDragZone.onDragDrop (ext-all.js:21)
    at Object.fireEvents (ext-all.js:21)
    at Object.fireEvents (ext-all.js:21)
```

### Environment
MODX 3
Google Chrome 79.0.3945.130

**Comment by JoshuaLuckers** [comment]
What version of MODX 3 are you using?

**Comment by CrazyBoy49z** [comment]
@JoshuaLuckers under the logo I see MODX alpha2

**Comment by JoshuaLuckers** [comment]
> @JoshuaLuckers under the logo I see MODX alpha2

Thanks, I believe I should get glasses haha.

**Comment by Ibochkarev** [comment]
@modxbot close
Fixed #14997
